torch/_torch_docs.py

```diff
@@ -1981,6 +1981,30 @@ def merge_dicts(*dicts):
     tensor([-1.,  1., -1., -1.])
 """.format(**common_args))
 
+add_docstr(torch.floor_divide,
+           r"""
+floor_divide(input, other) -> Tensor
+
+Each element of the tensor ``input`` is divided by each element of the tensor
```

Comment:
This is describing the implementation, not the result.  And the implementation is bad (not your fault), i.e. we have one floating point, one integer, we type promote to floating point, do floating point division, then truncate.  We probably shouldn't advertise that.  See numpy documentation for a reasonable wording.